### PR TITLE
[one-cmds] Add verbose option to tf2tfliteV2 command

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit changes the utils file to add the verbose option to tf2tfliteV2.

Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/utils.py [hunk 1]
```diff
@@ -138,6 +138,9 @@ def _parse_cfg(args, driver_name):
 def _make_tf2tfliteV2_cmd(args, driver_path, input_path, output_path):
     """make a command for running tf2tfliteV2.py"""
     cmd = [sys.executable, os.path.expanduser(driver_path)]
+    # verbose option
+    if args.verbose:
```
Comment (mhs4670go) [inline compiler/one-cmds/utils.py:142]: How about use this function for consistency?
```suggestion
    if _is_valid_attr(args, 'verbose'):
```

Comment (jyoungyun) [reply compiler/one-cmds/utils.py:142]: Sure.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**